WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -19171,6 +19352,7 @@
 				C737554027C80F1300C6E9A1 /* String+CondenseWhitespace.swift in Sources */,
 				7335AC6D21220F0F0012EF2D /* FormattableUserContent.swift in Sources */,
 				7335AC6221220E690012EF2D /* FormattableContentFactory.swift in Sources */,
+				09DBEA55281336E10019724E /* AppLocalizedString.swift in Sources */,
```

Comment:
This is me adding that `AppLocalizedString.swift` file (the one containing the implementation for the `AppLocalizedString` function) to the `WordPressNotificationServiceExtension` target, so I can call the function [in `WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift` below](https://github.com/wordpress-mobile/WordPress-iOS/pull/18427/files#diff-ef691831692bb20247ffbad1a2095c882f7871b8adb29e9d59d1a1f8900825a6R332)